Lob-API-public.yml

```
@@ -111,8 +111,33 @@ tags:
   - name: Intl Verifications
     description: Address verification for non-US addresses
 
+  - name: Intl Verifications Test Environment
+    x-traitTag: true
+    description: |
+      When verifying international addresses, you'll likely want to test against
+      a wide array of addresses to ensure you're handling responses correctly.
+      With your test API key, requests that use specific values for `primary_line`
+      let you explore the responses to many types of addresses:
+
+      | DELIVERABILITY OF SAMPLE RESPONSE | SET `primary_line` TO    |
+      |-----------------------------------|--------------------------|
+      | `deliverable`                     | deliverable              |
+      | `deliverable_missing_info`        | deliverable missing info |
+      | `undeliverable`                   | undeliverable            |
+      | `no_match`                        | no match                 |
+
+      See the `test` request & response examples under [Intl Verification Examples](#operation/intl_verification) within the
+      "Verify an international address section" in Intl Verifications.
+
+      You can rely on the response from these examples generally matching the response
+      you'd see in the live environment with an address of that type (excluding the recipient field).
```

```suggestion
      you'd see in the live environment with an address of that type (excluding the `recipient` field).
```

Following the format on docs.lob.com...but I'm starting to feel like a schlong so...we can include this in one of my PRs 😅